SYSDB: remove index on `dataExpireTimestamp`

alexey-tikhonov opened the pull request:
This index was only used in cleanup tasks that don't run often. On the other hand, this index is huge and degrades performance on libldb in general.

alexey-tikhonov commented:
Partially addresses https://issues.redhat.com/browse/RHEL-12109

aplopez reviewed (approved):
LGTM.

sumit-bose reviewed (approved):
Hi,

thank you for the patch, ACK.

bye,
Sumit

thalman reviewed (approved):
ACK, thanks

alexey-tikhonov commented:
Pushed PR: https://github.com/SSSD/sssd/pull/7455

* `master`
    * f0d45464cee1d2a6a2719dbffe5bbf6189d0554a - SYSDB: remove index on `dataExpireTimestamp`
    * fc2a26c306e51b66680aef85aa0d2c41d8049a7f - TS_CACHE: never try to upgrade timestamps cache
    * b1ce55a91a42e708c1eb54d4f34f5c3e5eb36e88 - DEBUG: added missing newline
* `sssd-2-9`
    * 13e3d0390b9aaf72a855b857857c3cdd6eb6252a - SYSDB: remove index on `dataExpireTimestamp`
    * 09b23e78806d8930c3f1b9e411dc8cf464c18998 - TS_CACHE: never try to upgrade timestamps cache
    * 9af9ca123c7b62d7f5fc7c3c135d3e6b8553ebec - DEBUG: added missing newline
